platform/bukkit/src/main/java/me/lokka30/treasury/plugin/bukkit/hooks/miniplaceholders/MiniPlaceholdersHook.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * This file is/was part of Treasury. To read more information about Treasury such as its licensing, see <https://github.com/ArcanePlugins/Treasury>.
+ */
+
+package me.lokka30.treasury.plugin.bukkit.hooks.miniplaceholders;
+
+import io.github.miniplaceholders.api.Expansion;
+import java.io.File;
+import java.util.Locale;
+import java.util.UUID;
+import me.lokka30.treasury.plugin.bukkit.BukkitTreasuryPlugin;
+import me.lokka30.treasury.plugin.bukkit.vendor.BukkitVendor;
+import me.lokka30.treasury.plugin.core.hooks.Hook;
+import me.lokka30.treasury.plugin.core.hooks.PlayerData;
+import me.lokka30.treasury.plugin.core.hooks.placeholder.BasicPlaceholderExpansion;
+import me.lokka30.treasury.plugin.core.hooks.placeholder.PlaceholdersConfig;
+import me.lokka30.treasury.plugin.core.hooks.placeholder.minipspecific.MiniPlaceholdersConfig;
+import net.kyori.adventure.identity.Identity;
+import net.kyori.adventure.text.minimessage.tag.Tag;
+import org.bukkit.entity.Player;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public class MiniPlaceholdersHook implements Hook {
+
+    private BasicPlaceholderExpansion basicExpansion;
+
+    @Override
+    public @NotNull String getPlugin() {
+        return "MiniPlaceholders";
+    }
+
+    @Override
+    public boolean register() {
+        PlaceholdersConfig config = MiniPlaceholdersConfig.load(new File(
+                ((BukkitTreasuryPlugin) BukkitTreasuryPlugin.getInstance()).dataFolder(),
+                "miniPlaceholders_config.yml"
```

Comment:
This file name contains multiple formatting choices - camel case and snake case. It also has no purpose other than to configure the hook. In that case, let's go for this instead:
```suggestion
                "miniplaceholders-hook.yml"
```